### PR TITLE
feature/test2 1st commit

### PR DESCRIPTION
**Pull request by petrushjlee**
i need your review

### CODE REVIEW

File: sub.c [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#include <math.h>
+
+int sub(int a, int b) {
```
Comment (petrushjlee) [inline sub.c:3]: comment ...

File: sub.c [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#include <math.h>
+
```
Comment (petrushjlee) [inline sub.c:2]: ??

File: sub.c (hunk 1 above)
Comment (justin-labry) [reply sub.c:3]: Please merge to develop

File: sub.c (hunk 2 above)
Comment (petrushjlee) [reply sub.c:2]: aaaa

File: main.c [hunk 3]
```diff
@@ -7,4 +7,8 @@ int main()
 aaaaa
 bbbbbb
 ccccc
+
```
Comment (WonjooPARK) [inline main.c:10]: + new line.

File: main.c [hunk 4]
```diff
@@ -7,4 +7,8 @@ int main()
 aaaaa
 bbbbbb
 ccccc
```
Comment (WonjooPARK) [inline main.c:9]: +new review

**Review by justin-labry — APPROVED** [review]
Very good!

**Review by justin-labry — APPROVED** [review]
This is good!

**Review by justin-labry — APPROVED** [review]
Awesome!

**Review by justin-labry — APPROVED** [review]
Finish

**Review by petrushjlee — COMMENTED** [review]
finished

**Comment by justin-labry** [comment]
This is ok. I will merge.

**Pull request closed without merging**